src/hooks/index.js

```diff
@@ -644,25 +769,36 @@ exports.checkSchoolOwnership = (context) => {
 	const genericService = context.app.service(service);
 	const userService = context.app.service('users');
 
-	return Promise.all([userService.get(userId), genericService.get(objectId)])
-		.then((res) => {
-			if (res[0].schoolId.equals(res[1].schoolId)) return context;
-			throw new Forbidden('You do not have valid permissions to access this.');
-		});
+	return Promise.all([
+		userService.get(userId),
+		genericService.get(objectId),
+	]).then((res) => {
+		if (res[0].schoolId.equals(res[1].schoolId)) return context;
+		throw new Forbidden(
+			'You do not have valid permissions to access this.',
+		);
+	});
 };
 
 function validatedAttachments(attachments) {
 	let cTotalBufferSize = 0;
 	attachments.forEach((element) => {
-		if (!element.mimetype.includes('image/')
-			&& !element.mimetype.includes('video/')
-			&& !element.mimetype.includes('application/msword')
-			&& !element.mimetype.includes('application/pdf')) {
+		if (
+			!element.mimetype.includes('image/') &&
+			!element.mimetype.includes('video/') &&
+			!element.mimetype.includes('application/msword') &&
+			!element.mimetype.includes('application/pdf')
+		) {
 			throw new Error('Email Attachment is not a valid file!');
 		}
 		cTotalBufferSize += element.size;
-		if (cTotalBufferSize >= MAXIMUM_ALLOWABLE_TOTAL_ATTACHMENTS_SIZE_BYTE) {
-			throw new BadRequest('Email Attachments exceed the max. total file limit.');
+		if (
+			cTotalBufferSize >=
```

Comment:
Codacy found an issue: ['>=' should be placed at the beginning of the line.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5994445)